scalac-scoverage-plugin/src/main/scala/scoverage/plugin.scala

```diff
@@ -35,16 +34,33 @@ class ScoveragePlugin(val global: Global) extends Plugin {
         options.dataDir = opt.substring("dataDir:".length)
       } else if (opt.startsWith("extraAfterPhase:") || opt.startsWith("extraBeforePhase:")) {
         // skip here, these flags are processed elsewhere
-      } else {
+      } else if (opt.startsWith("classpathSubdir:")){
+        options.classpathSubdir = opt.substring("classpathSubdir:".length)
+      }else if (opt.startsWith("writeToClasspath:")) {
+        try {
+          options.writeToClasspath = opt.substring("writeToClasspath:".length).toBoolean
+        } catch {
+          case _: IllegalArgumentException => throw new RuntimeException("writeToClasspath can only be a boolean value: true or false")
+          case e: Exception => throw e
+        }
+      } else{
         error("Unknown option: " + opt)
       }
     }
-    if (!opts.exists(_.startsWith("dataDir:")))
-      throw new RuntimeException("Cannot invoke plugin without specifying <dataDir>")
+
+    if (opts.exists{p: String => p.contains("writeToClasspath") && p.contains("false")} || (!opts.exists(_.startsWith("writeToClasspath")))) {
+      if (!opts.exists(_.startsWith("dataDir:")))
+        throw new RuntimeException("Cannot invoke plugin without specifying <dataDir>")
+    }
     instrumentationComponent.setOptions(options)
   }
 
   override val optionsHelp: Option[String] = Some(Seq(
+    "-P:scoverage:writeToClasspath:<boolean>               if true, use the environment variable to store measurements" +
+                                                           "and store instruments to the output classpath where compiler emits classfiles." +
+                                                           "Overrides the datadir with the output classpath.",
+    "-P:scoverage:classpathSubdir:<pathtosubdir>           subdir to attach to classpath. Default: META-INF/scoverage",
```

Comment:
This should reference the other option probably: "subdir to prefix to classpath when `writeToClasspath` is in use"